team/bundles/org.eclipse.core.pki/plugin.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
missed license header

Reply:
Can I grab a header off file already in a bundle, copy and paste to all files needing header????